lib/mysql_rewinder/cleaner.rb

```diff
@@ -31,5 +35,22 @@ def all_tables
         WHERE TABLE_SCHEMA = DATABASE()
       SQL
     end
+
+    private
+
+    def log_sql(sql)
+      return yield unless @logger.debug?
+
+      start_ts = Time.now
+      res = yield
+      duration = (Time.now - start_ts) * 1000
+
+      name = "Cleaner SQL (#{duration.round(1)}ms)"
```

Comment:
Add some message like the following, so that users can easily distinguish logs of MysqlRewinder from other query logs.

```suggestion
      name = "[MysqlRewinder] Cleaner SQL (#{duration.round(1)}ms)"
```